admin/app/components/solidus_admin/products/show/component.html.erb

```diff
@@ -0,0 +1,110 @@
+<div class="<%= stimulus_id %> px-4 relative" data-controller="<%= stimulus_id %>">
+  <header class="py-6 flex items-center gap-4">
+    <%= render component('ui/button').new(
+      tag: :a,
+      alt: t('.back'),
+      icon: 'arrow-left-line',
+      scheme: :secondary,
+      href: solidus_admin.products_path,
+    ) %>
+    <h1 class="flex items-center gap-2">
+      <span class="body-title"><%= @product.name %></span>
+      <%= render component('products/status').new(product: @product) %>
+    </h1>
+
+    <div class="ml-auto flex gap-2 items-center">
+      <%= render component('feedback').new %>
+      <%= render component('ui/button').new(
+        tag: :a,
+        text: t('.duplicate'),
+        href: spree.clone_admin_product_path(@product),
+        scheme: :ghost,
+      ) %>
+      <%= render component('ui/button').new(
+        tag: :a,
+        text: t('.view'),
+        href: SolidusAdmin::Config.storefront_product_path(@product),
+        target: :_blank,
+        scheme: :ghost,
+      ) %>
+      <%= render component('ui/button').new(
+        tag: :button,
+        text: t('.save'),
+        form: form_id,
+      ) %>
+    </div>
+  </header>
+
+  <%= form_for @product, url: product_path(@product), html: { id: form_id } do |f| %>
```

Comment:
FYI, we added the product component to lay down the architecture to define forms from the outside, which would later enable a data layer on top of it. Please see https://github.com/solidusio/solidus/pull/5360